script.js

```diff
@@ -74,8 +74,8 @@ performerButton.setAttribute('type', 'button');
 performerButton.innerHTML = 'MORE <i class="fas fa-angle-down"></i>';
 performerButton.addEventListener('click', () => {
   document.querySelector('.speakers-group').innerHTML = '';
-  let stagePerformerLength = stagePerformer.length;
-  for (let i = 0; i < stagePerformerLength; i++) {
+  let speakersLength = speakers.length;
+  for (let i = 0; i < speakersLength; i++) {
     const performer = document.createElement('article');
     const performerImage = document.createElement('div');
     const imageDots = document.createElement('img');
```

Comment:
- [ ] (OPTIONAL) For the speakers' section, kindly make sure the `More` button displays only on the mobile view. In the desktop view, all the speakers are to be displayed but in the mobile view, only 2 speakers are displayed together with the `More` button. Kindly address this issue 👍 
- Your design:
![Annotation 2022-04-23 052946](https://user-images.githubusercontent.com/85113567/164877849-caee7a32-d004-44c3-bb89-98bae259f7c3.jpg)

Reply:
I'll make the change. Thank you for your nice review.

Reply:
Make sure to keep the functionality of the `More` button in the mobile view 👍

Reply:
Sure.